qqq-backend-core/src/main/java/com/kingsrook/qqq/backend/core/model/audits/AuditsMetaDataProvider.java

```diff
@@ -176,7 +176,7 @@ private QTableMetaData defineAuditTableTable(String backendName)
          .withRecordLabelFields("label")
          .withPrimaryKeyField("id")
          .withUniqueKey(new UniqueKey("name"))
-         .withField(new QFieldMetaData("id", QFieldType.INTEGER))
+         .withField(new QFieldMetaData("id", QFieldType.LONG))
```

Comment:
i mean, i doubt we'll ever have more than 2 billion tables or users, so i probably wouldn't have changed those to longs myself, but, i suppose why not?  kinda?  small tables, so the extra 4-bytes for those 100 rows doesn't matter.  but then "inconsistent" with the rest of the system where all ids are integers... but otherwise inconsistent w/ the rest of the audit "subsystem" with longs... hmm, yeah, so, yeah!